<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="mainStackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>691</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QWidget" name="gameViewWidget">
     <property name="styleSheet">
      <string notr="true">QProgressBar::chunk {
     background-color: red;
     width: 1px;
 }

 QProgressBar {
     border: 2px solid black;
     border-radius: 0px;
     text-align: center;
 }

QPushButton {
	background-color : white;
	border : 2px solid black;
	border-radius :7;
}

QPushButton::Pressed {
	background-color : gray;
	border-style : inset;
}</string>
     </property>
     <widget class="QStackedWidget" name="gameStackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>691</width>
        <height>541</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="medicalView">
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>240</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>medicalView</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="mapView">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>159</x>
          <y>9</y>
          <width>521</width>
          <height>521</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="mapViewLayout"/>
       </widget>
       <zorder>commonGameWidget</zorder>
       <zorder>commonGameWidget</zorder>
       <zorder>commonGameWidget</zorder>
       <zorder>commonGameWidget</zorder>
       <zorder>commonGameWidget</zorder>
       <zorder>gridLayoutWidget</zorder>
      </widget>
      <widget class="QWidget" name="craftingView">
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>260</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>craftingView</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="itemView">
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>250</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>itemView</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="commonGameWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>691</width>
        <height>541</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Thirst</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="rest">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>61</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Core Temp.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>AP :</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="thirst">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QProgressBar" name="ambientTemperature">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="actionPoints">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>16</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QPushButton" name="craftingButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>470</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Crafting</string>
       </property>
      </widget>
      <widget class="QPushButton" name="itemsButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Items</string>
       </property>
      </widget>
      <widget class="QPushButton" name="homeButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="coreTemperature">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="medicalButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>420</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Medical</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="hunger">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>71</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Amb. Temp.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Hunger</string>
       </property>
      </widget>
      <widget class="QFrame" name="Actions">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>141</width>
         <height>161</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <widget class="QLabel" name="actions">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Actions</string>
        </property>
       </widget>
       <widget class="QPushButton" name="action1Button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="action2Button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="action3Button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="action4Button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Rest</string>
       </property>
      </widget>
      <widget class="QPushButton" name="mapbutton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Map</string>
       </property>
      </widget>
      <zorder>label_3</zorder>
      <zorder>rest</zorder>
      <zorder>label_5</zorder>
      <zorder>label</zorder>
      <zorder>thirst</zorder>
      <zorder>ambientTemperature</zorder>
      <zorder>actionPoints</zorder>
      <zorder>craftingButton</zorder>
      <zorder>itemsButton</zorder>
      <zorder>homeButton</zorder>
      <zorder>coreTemperature</zorder>
      <zorder>medicalButton</zorder>
      <zorder>hunger</zorder>
      <zorder>label_6</zorder>
      <zorder>label_2</zorder>
      <zorder>Actions</zorder>
      <zorder>label_4</zorder>
      <zorder>mapbutton</zorder>
      <zorder>tableView</zorder>
      <zorder>mapViewWidget</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="menuViewWidget">
     <widget class="QStackedWidget" name="menuStackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>691</width>
        <height>541</height>
       </rect>
      </property>
      <widget class="QWidget" name="homeView">
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>220</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>homeView</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="setupView">
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>250</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>setupView</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <zorder>mainStackedWidget</zorder>
   <zorder>mapViewWidget</zorder>
   <zorder>commonGameWidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
